src/main/java/hudson/plugins/warnings/WarningsDescriptor.java

```diff
@@ -4,12 +4,7 @@
 import java.util.Collection;
 
 import org.jenkinsci.Symbol;
-import org.kohsuke.stapler.Ancestor;
-import org.kohsuke.stapler.AncestorInPath;
-import org.kohsuke.stapler.QueryParameter;
-import org.kohsuke.stapler.StaplerProxy;
-import org.kohsuke.stapler.StaplerRequest;
-import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.*;
```

Comment:
Please do not do that in production code. It may cause ambiguity in some edge cases